src/main/java/com/liveramp/daemon_lib/DaemonCallback.java

```diff
@@ -0,0 +1,17 @@
+package com.liveramp.daemon_lib;
+
+import com.liveramp.daemon_lib.utils.DaemonException;
+
+public interface DaemonCallback {
+
+  void callback() throws DaemonException;
+
+  class None implements DaemonCallback {
+
+    @Override
+    public void callback() throws DaemonException {
+
+    }
+  }
+
+}
```

Comment:
Wakeup callback has nothing to do with joblet configs, so we need this new generic callback interface